### PR TITLE
Calling code

### PR DESCRIPTION
**Pull request by henrymalik19**
I added the logic for country.js and split th API calles into utl/country.js
Added a country option to switch in index.js
Added another switch option in the location.js file so that I could use that module to get current country data.

May suggest refactoring that abit so that redundant data is not returned. 
ie return one object and let the calling module use the data that is needed.

### CODE REVIEW

File: tests/unit/country.test.js [hunk 1]
```diff
@@ -0,0 +1 @@
+const country = require("./cmds/country");
```
Comment (drum-IT) [inline tests/unit/country.test.js]: Looks like this file path is incorrect. Tests are failing.

Comment (henrymalik19) [reply tests/unit/country.test.js]: hmm weird thats the only thing in the file. Could you help me with creating a test file. I am a bit confused

Comment (drum-IT) [reply tests/unit/country.test.js]: Talked this through with Malik over Slack. He is going to work on a test script.

**Review by drum-IT — APPROVED** [review]
Update the file path in your test file, and things should

**Pull request merged**